manageiq-operator/Dockerfile

```
@@ -1,5 +1,18 @@
 # Build the manager binary
-FROM docker.io/library/golang:1.23 as builder
+FROM registry.access.redhat.com/ubi9/ubi-minimal as builder
+
+COPY go.mod /go.mod
+
+RUN microdnf -y install gzip jq tar && \
+    arch=$(uname -m) && \
+    if [ ${arch} == "x86_64" ] ; then \
+      arch="amd64" \
+    ; fi && \
```

You can inline this and I think it's will look nicer

```suggestion
    if [ ${arch} == "x86_64" ] ; then arch="amd64"; fi && \
```